### DISCUSSION TITLE
Ref in v-for. nuxt3

### DISCUSSION THREAD

**Discussion by asifmushtaqse** [nuxt/nuxt#17840] [Questions]
```
<script setup lang="ts">

    var { movies } = useMovies()
    var itemRefs = ref([])
    onMounted(() => {
        console.log(itemRefs.value[0])
    })
</script>

<template>
    <section class="container overflow-hidden">
        
                <ul>
                    <li v-for="(movie, index) in movies" ref="itemRefs">
                        
                    </li>
                </ul>
    </section>
</template>
```

In the above code, I'm following the convention from this link. [Refs in v-for](https://vuejs.org/guide/essentials/template-refs.html#refs-inside-v-for). But still it is giving undefined on the console.
I'm not sure it is bug, or I'm doing something wrong?

I'm using nuxt3.

**Comment by danielroe** [comment]
This is a Vue 3 bug. There are a couple of issues in the core repo, but this might be one: https://github.com/vuejs/core/issues/5525
